templates/web/brent/waste/garden/subscribe_intro.html

```diff
@@ -0,0 +1,62 @@
+[% SET bodyclass = 'waste' %]
+
+[% INCLUDE header.html %]
+
+  [% PROCESS 'govuk/fields.html' %]
+  [% PROCESS title title=form.title %]
+  [% PROCESS errors %]
+  <p>
+    Have an intro here
+  </p>
+
+  <h3>How much does it cost?</h3>
+
+  <p>
+    The Brent Garden Waste service runs from 1st April - 31st March and costs £60.
+
+    If you sign up to the service from the 1st October a discount will apply. You will be charged £30 for the service until the end of March (a 6 month period). If you are in receipt of certain benefits, you are entitled to a 20 percent discount on the total cost of the garden waste service.
```

Comment:
Based on comments on Basecamp, presumably will add a note to say call if you want to get this discount (for now) - they can confirm wording.